docs/advanced_examples/LogisticRegression.ipynb

```diff
@@ -450,6 +451,174 @@
     "Decision boundaries from both models are almost identical. This shows executing a logistic regression\n",
     "model in FHE doesn't really impact the performances compared to its scikit-learn equivalent.  "
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "80d8ac5f-a78e-45c7-9251-5a37476d4768",
+   "metadata": {},
+   "source": [
+    "The plot above shows that the quantization introduced by Concrete ML didn't degrade the performance of the model.\n",
```

Comment:
this section is awesome ! I would just add a title to it so that we quickly get what the following is about